safe-global/web-core
# fix: Add owner icon to confirmations

## What it solves

Resolves #1279 

## How this PR fixes it

- Displays the owner icon for transactions on the dashboard 
- Uses a lighter green for the confirmations background in the dashboard according to the design
- Changes owner icon color in the queue to be consistent with the text color 

## How to test it

1. Open a safe with a queued transaction
2. Navigate to the dashboard
3. Observe the owner icon next to the number of confirmations
4. Navigate to the Queue
5. The owner icon should have the same color as the text next to it

## Screenshots
<img width="632" alt="Screenshot 2022-12-22 at 14 14 08" src="https://user-images.githubusercontent.com/5880855/209142003-5d629cb2-db9b-4be1-b5c6-82314dd66e3b.png">
<img width="628" alt="Screenshot 2022-12-22 at 14 15 48" src="https://user-images.githubusercontent.com/5880855/209142247-4f054b12-9da3-48e6-a4e1-ea6f14037a5c.png">
<img width="798" alt="Screenshot 2022-12-22 at 14 15 55" src="https://user-images.githubusercontent.com/5880855/209142265-8814bbbc-d4a9-4230-919c-9df350297610.png">
